stripe/res/values-DA/strings.xml

```diff
@@ -15,7 +15,6 @@
     <string name="address_label_address">Adresse</string>
     <string name="address_label_address_line1">Adresselinje 1</string>
     <string name="address_label_address_line1_optional">Adresselinje 1 (valgfrit)</string>
-    <string name="address_label_address_line2">Adresselinje 2</string>
```

Comment:
Why is this removed?

Reply:
It's not used.

Reply:
"address_label_address_line2_optional" is the one that's always used.